Mathlib/SetTheory/Surreal/Ordinal.lean

```diff
@@ -0,0 +1,37 @@
+/-
+Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.SetTheory.Game.Ordinal
+import Mathlib.SetTheory.Surreal.Basic
+
+/-!
+# Surreals as games
+
+We define the canonical map `Ordinal → Surreal` in terms of the map `Ordinal.toPGame`.
+
+# Main declarations
+
+- `Ordinal.toSurreal`: The canonical map between ordinals and surreal numbers.
+-/
+
+open Surreal SetTheory PGame
+
+open scoped NaturalOps
```

Comment:
Do you use this?

Reply:
Oops, not yet, this was a left-over from #21979.